matrix-sdk/src/main/java/org/matrix/androidsdk/db/MXMediasCache.java

```diff
@@ -294,23 +295,20 @@ public static void clearThumbnailsCache(Context applicationContext) {
     }
 
     /**
-     * Convert matrix url into http one.
+     * Resolve a Matrix media content URI (in the form of "mxc://...") into an HTTP URL.
      *
-     * @param url    the matrix url
+     * @param url    the matrix url, must not be null.
```

Comment:
Why it must not be null? The nullity checks are done.

Reply:
you're right, I will remove this comment